internal/core/labels.go

```diff
@@ -21,3 +25,20 @@ func DefaultLabels(sessionID string) map[string]string {
 		LabelVersion:   internal.Version,
 	}
 }
+
+// MergeCustomLabels merges default labels in-place to the custom labels.
+//
+// It is an error to use "org.testcontainers" as a prefix to custom labels.
+func MergeCustomLabels(customLabels map[string]string, defaultLabels map[string]string) error {
+	for customLabelKey := range customLabels {
+		_, present := defaultLabels[customLabelKey]
+		if present || strings.HasPrefix(customLabelKey, LabelBase) {
+			return fmt.Errorf("custom labels cannot begin with %s or already be present", LabelBase)
```

Comment:
Do we want to return an error here, or simply ignore those keys?

Reply:
I don't have an opinion. I did the same thing for python. https://github.com/testcontainers/testcontainers-python/blame/0ce4fecb2872620fd4cb96313abcba4353442cfd/core/testcontainers/core/labels.py#L19

What does Java do?

Reply:
I think it's more user-friendly not returning an error here and possibly inform the user in the logs: "you did pass a wrong prefixed key? I can continue without them" Vs "you did pass a wrong prefixed key? Please stop and fix that". I know the latter is closer to what the Go compiler would say, right? So probably changing my mind while typing. Wdyt?